distribution/changelog.txt

```diff
@@ -63,6 +62,7 @@
 - Fix: [#8585] Part of track missing on air powered vertical coaster.
 - Fix: [#8588] Guest list scrolling breaks above ~2000 guests.
 - Fix: [#8591] Game loop does not run at a consistent tick rate of 40 Hz.
+- Fix: [#8647] Increased 'guest generation probability' thresholds of marketing campaigns (original bug). 
```

Comment:
Oh ****, forgot to look at the message proper. This is the last time, I promise! (Reason for change: we always tell which bug is fixed, not _how_ it was fixed.)
```suggestion
- Fix: [#8647] Marketing campaigns check for entry fees below £1 (original bug). 
```